charts/app-config/values-staging.yaml

```diff
@@ -630,6 +630,12 @@ govukApplications:
       dbMigrationEnabled: true
       nginxClientMaxBodySize: &max-upload-size 500M
       workerEnabled: true
+      redis:
+        enabled: true
+        redisUrlOverride:
+          app: &content-publisher-redis >
```

Comment:
I'm assuming we don't need the `-` here after all?

Reply:
It seems to make no difference, the correct value gets populated.

My issue yesterday turned out to be completely unrelated to my changes.